src/staging/__tests__/components/__snapshots__/footer.test.js.snap

```diff
@@ -81,26 +52,46 @@ exports[`Footer renders correctly 1`] = `
                     </div>
                     <div
                       className="footer-menu-divider"
+                    />
+                  </li>
+                  <li>
+                    <div
+                      className="mt-2"
                     >
-                       
+                      <a
+                        href="/commercial-use"
+                      >
+                        Commercial use
+                      </a>
                     </div>
                   </li>
                   <li>
                     <div
                       className="mt-2"
                     >
                       <a
-                        href="/commercial-use"
+                        href="/commercial-use/filming-in-parks"
```

Comment:
I think we're going with training slashes for internal urls, as this (and the others) currently re-direct. We'll have that chat next week to ensure that's correct.

Reply:
Sorry, I forgot to ask you about this. Do you know whether we should keep it as is or not?

Reply:
Oh, leave it for now. I'll make a ticket to ensure we discuss it later.